src/Classes/Options/Interactivity/Modes/Connect.ts

```diff
@@ -17,6 +20,8 @@ export class Connect implements IConnect {
      * @param value
      */
     public set line_linked(value: IConnectLineLinked) {
+        Messages.deprecationMessage("interactivity.modes.connect.line_linked", "interactivity.modes.connect.lineLinked");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Exceeds maximum line length of 120](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5139697)